js/main.js

```diff
@@ -31,6 +32,8 @@ require('./memoryMonitor.js');
 
 const windowMgr = require('./windowMgr.js');
 
+crashReporter.start({companyName: 'Symphony', submitURL: 'http://localhost:3000', uploadToServer: false, extra: {'process': 'main'}});
```

Comment:
why the http://localhost:3000???

Reply:
It's a default value. We can change it when we have the server up. Unfortunately, submitURL is a mandatory attribute.

Reply:
sounds good.